wiki/osu!stream/de.md

```diff
@@ -0,0 +1,254 @@
+
+##### Hold circle
+
+![Hold circle 1][PC.jpg]
+
+_Hold circle 1_
+
+-
+
+![Hold circle 2][OS-HC2.jpg]
+
+_Hold circle 2_
+
+- Halte den Kreis so lange, bis der Kreis "explodiert
+- 300 wenn perfekt gemacht.
+  - 100 für einen ungenaueren anfänglichen Halt und ein korrektes Halten.
+  - Zumindest 50 für das Berühren.
+  - 30 für jede Explosion.
+  - Multiplikator-Reset, wenn die Kreise nicht richtig gehalten wurden, und Miss, wenn sie dies nicht taten.
+
+**Sliders**
+
+- ** Die Beurteilung von Schiebereglern hängt von der Genauigkeit des anfänglichen Abgriffs ab **.
+- Schieberegler bestehen aus Schieberegler-Häkchen, die die Start-, End- und Wiederholungspunkte des Schiebereglers enthalten.
+  - Ein 300er, wenn alle Slider-Ticks und der genaue erste Tipp erfolgreich ausgeführt wurden.
+  - Eine 100, wenn mindestens die Hälfte von ihnen mit erheblichem Anfangsabgriff.
+  - Eine 50, wenn es gelungen ist, auch nur einen der Ticks oder den ersten Tipp zu treffen.
+- ** Ein zu früher Klick auf einen Schieberegler ** führt nicht zu einem Fehlschlag, ** setzt jedoch den Punktemultiplikator auf 0 zurück **.
+- Ein Fehlschlag, wenn der Schieberegler nicht funktioniert hat.
+
+**Spinner**
+
+- Für einen Spinner 300, 100 oder 50, abhängig von der Anzahl der ausgeführten Spins im Vergleich zur Länge des Spinners.
+- Ein Fehler, wenn der auf beiden Seiten des Spinners angezeigte Zähler nicht gefüllt werden kann.
+
+### Multiplikator
+
+Folgendes fügt dem Punktemultiplikator jeweils einen Punkt hinzu:
+
+- Ein erfolgreicher Treffer im Trefferkreis.
+- Ein erfolgreiches Halten des Haltekreises.
+- Ein erfolgreicher Treffer zu Beginn eines Schiebereglers.
+- Ein erfolgreich berührtes Ende eines Schiebereglers.
+- Ein erfolgreich auf den Slider getipptes Häkchen.
+- Ein erfolgreich abgeschlossener Spinner.
+
+Das Folgende setzt den Punktemultiplikator auf Null zurück:
+
+- Fehlender Trefferkreis.
+- Freigabe des Haltekreises, wenn dieser noch nicht aufgelöst ist.
+- Die verbundenen Beats können nicht gleichzeitig getroffen werden. (Hit eins verpassen eins)
+- Ein Klick auf einen Schieberegler startet viel zu früh.
+- Fehlender Slider-Start oder Slider-Tick.
+- Die Anzeige eines Spinners mit einem eindeutigen Status kann nicht aufgefüllt werden.
+
+Folgendes erhöht oder setzt den Punktemultiplikator nicht zurück:
+
+- Fehlendes Slider-Ende. (Wird zu einem guten Ergebnis führen! Oder 100/50 Treffer)
+- Spinner's Score Bonus
+
+### Noten
+
+! [Notenbildschirm] [Os-Grade.jpg]
+
+**Normale Noten**
+
+- SS = 100% Genauigkeit
+- S = Über 90% Genauigkeit
+- A = Über 80% Genauigkeit
+- B = Über 70% Genauigkeit
+- C = Über 60% Genauigkeit
+- D = Sonst noch was.
+
+### Lebensdaueranzeige
+
+Im Gegensatz zum osu! -Standard unterscheidet sich die Lebensleiste in osu! streamen, da es auf dem Gameplay-Modus basiert.
+
+Im **einfachen Modus wird die Lebensleiste vollständig entfernt**.
+
+![Beispiel für Stream Up! von Normal][Bild hoch streamen]

```

Comment:
```suggestion
![Beispiel für Stream Up! von Normal][Bild streamen]
```

This is a reference link, you have translated the text on the right twice but differently (from line 11). The two must match to work correctly.